avaframe/out3Plot/plotUtils.py

```diff
@@ -577,6 +577,8 @@ def putAvaNameOnPlot(ax, avaDir, date=True):
     ax.annotate(
         infoText,
         xy=(0.01, 0.01),
+        color=color,
+        fontsize=fontsize,
```

Comment:
Is fontsize=None actually the default?

Reply:
yes but If I want to change it needs to be a parameter as input..